patterns/1-initial/explicit-shared-ownership.md

```diff
@@ -0,0 +1,47 @@
+# Title
+
+Explicit Shared Ownership
+
+# Patlet
+
+In a software system there is a component that several teams depend on. Initially developed by a handful of people it has long outgrown it's orignal use-case. Original authors are no longer capable of taking full ownership of that component. Development happens organically. Often there is confusion as to who should make changes, who needs to approve changes. Making ownership explicit, sharing that ownership across dependent teams, making expected behaviour explicit helps remove ambiguity. Writing those learnings down means that it can be changed according to the same process that is used to change the software.
+
+# Problem
+
+An organisation is already using InnerSource best practices in several teams. The architecture of the software offered has grown organically. Talking about code ownership and accountability teams notice that there is a component that is in a worrying state: Developed by an original team of authors it has grown it's userbase way beyond what the original team can handle. From time to time others step up to help out, however there is no formal process established. As a result conflicts arise around who should do the work, who should decide on project direction. Simply forking the component would lead to a lot of duplication and wasted effort. As a result teams are looking for a less intrusive solution to the issue.
```

Comment:
```suggestion
An organisation is already using InnerSource best practices in several teams. The architecture of the software offered has grown organically. 

While talking about code ownership and accountability, teams notice that there is a component that is in a worrying state: Developed by an original team of authors it has grown it's userbase way beyond what the original team can handle. From time to time others step up to help out, however there is no formal process established. As a result, conflicts arise around who should do the work, and who should decide on project direction. 

Simply forking the component would lead to a lot of duplication and wasted effort. Therefore the involved teams are looking for a less intrusive solution to the issue.
```

Splitting up the paragraph into sections, and making some wording and comma fixes.